sssd_test_framework/utils/authentication.py

```diff
@@ -22,6 +23,15 @@
 DEFAULT_AUTHENTICATION_TIMEOUT: int = 60
 """Default timeout for authentication failure."""
 
+class PasskeyAuthenticationUseCases(Enum):
+    """
+    Authentication methods for passkey authentication.
+    """
+    PASSKEY_WITH_PIN = 0,
+    PASSKEY_WITH_PIN_AND_PROMPTS = 1,
+    PASSKEY_WITHOUT_PIN_WITH_PROMPTS = 2,
+    PASSKEY_WITHOUT_PIN_WITHOUT_PROMPTS = 3,
+    PASSKEY_FALLBACK_TO_PASSWORD = 4,
```

Comment:
In my opinion these names are more clear.

PASSKEY_PIN
PASSKEY_PIN_AND_PROMPTS
PASSKEY_PROMPTS_NO_PIN
PASSKEY_NO_PIN_NO_PROMPTS
PASSKEY_FALLBACK_TO_PASSWORD

Reply:
updated.